units/crons.go

```diff
@@ -1230,3 +1231,42 @@ func PopulateSSHKeyUpdates(env evergreen.Environment) amboy.QueueOperation {
 		return catcher.Resolve()
 	}
 }
+
+func PopulateReauthorizationJobs(env evergreen.Environment) amboy.QueueOperation {
+	return func(ctx context.Context, queue amboy.Queue) error {
+		if !env.UserManagerInfo().CanReauthorize {
+			return nil
+		}
+
+		flags, err := evergreen.GetServiceFlags()
+		if err != nil {
+			return errors.WithStack(err)
+		}
+		if flags.BackgroundReauthDisabled {
+			grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
+				"message": "background reauth is disabled",
+				"impact":  "users will reauth on page loads after periodic auth expiration",
+				"mode":    "degraded",
+			})
+			return nil
+		}
+
+		reauthAfter := time.Duration(env.Settings().AuthConfig.BackgroundReauthMinutes) * time.Minute
+		if reauthAfter == 0 {
+			reauthAfter = defaultBackgroundReauth
+		}
+		users, err := user.FindNeedsReauthorization(reauthAfter)
+		if err != nil {
+			return err
+		}
+
+		catcher := grip.NewBasicCatcher()
+		ts := util.RoundPartOfMinute(20).Format(TSFormat)
+		for _, user := range users {
+			catcher.Wrap(env.RemoteQueue().Put(ctx, NewReauthorizationJob(env, &user, ts)), "could not enqueue jobs to reauthorize users")
```

Comment:
What's the reason for putting the job in both queues?

Reply:
My bad, this was from local testing. I removed it.